packages/drivers/odsp-driver/src/odspPublicUtils.ts

```diff
@@ -6,6 +6,8 @@
 import { hashFile, IsoBuffer } from "@fluid-internal/client-utils";
 import { ISequencedDocumentMessage, ISnapshotTree } from "@fluidframework/protocol-definitions";
 
+// TODO: add documentation
```

Comment:
If anyone has suggestions for good semantic docs for this, I'm happy to add them in this PR. Otherwise, they can be added later.